holoviews/plotting/mpl/tabular.py

```diff
@@ -89,7 +89,7 @@ def pprint_value(self, value):
         elif isinstance(value, str):
             formatted = safe_unicode(value)
         else:
-            formatted = str(value)
```

Comment:
Hmm, changing the previous block with:

```
elif isinstance(value, basestring):
```

where basestring is import from holoviews.core.util, should do it.